gammapy/estimators/map/core.py

```diff
@@ -1138,6 +1146,16 @@ def slice_by_coord(self, slices):
         -------
         flux_maps : `FluxMaps`
             Sliced flux maps object.
+
+        Examples
+        --------
+        >>> from gammapy.estimators import FluxPoints
+        >>> import astropy.units as u
+        >>> lc_1d = FluxPoints.read(
+        ...     "$GAMMAPY_DATA/estimators/pks2155_hess_lc/pks2155_hess_lc.fits", format="lightcurve"
+        ... )
+        >>> slices = {"time": slice(2035.93*u.day, 2036.05*u.day)}
+        >>> sliced = lc_1d.slice_by_coord(slices)
```

Comment:
Maybe show a simultaneous slicing on energy and time

Reply:
This light curve only have one energy bin, otherwise I would have